recipe/emailpassword/epmodels/models.go

```diff
@@ -27,7 +27,7 @@ type TypeNormalisedInput struct {
 	ResetPasswordUsingTokenFeature TypeNormalisedInputResetPasswordUsingTokenFeature
 	EmailVerificationFeature       evmodels.TypeInput
 	Override                       OverrideStruct
```

Comment:
You have to mark `createAndSendCustomEmail` as deprectaed in the type input for reset password